src/validator.ts

```diff
@@ -26,24 +26,30 @@ export class Validator {
         const validate = ajv.compile(schema);
         const valid = validate(gitLabCiConfig);
         if (!valid) {
-            writeStreams.stderr(chalk`Invalid .gitlab-ci.yml configuration!\n`);
-
             const betterErrors = betterAjvErrors({
                 data: gitLabCiConfig,
                 errors: validate.errors,
             });
 
-            const errors = betterErrors.map((e => (chalk`\t• {redBright ${e.message}} at {blueBright ${e.path}}`)));
-            writeStreams.stderr(errors.splice(0, MAX_ERRORS).join("\n"));
-            if (errors.length > MAX_ERRORS) {
-                writeStreams.stderr(`\t... and ${errors.length - MAX_ERRORS} more`);
```

Comment:
seemed like there's some bug here, should be using `betterErrors.length` instead of `errors.length`

since `splice` in line 37 mutated the length